### PR TITLE
[ci] Enable Style/IndentArray

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

**Comment by DavidKang** [comment]
[Enable the Style/IndentArray cop in rubocop](https://trello.com/c/mqgi05jw/195-enable-the-style-indentarray-cop-in-rubocop)

**Comment by Ana06** [comment]
Rebase! :wink:

File: src/api/.rubocop.yml [hunk 1]
```diff
@@ -65,6 +65,14 @@ Style/HashSyntax:
   Exclude:
     - 'config/routes.rb'
 
+# Checks the indentation of the first element in an array
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
+# SupportedStyles: special_inside_parentheses, consistent, align_brackets
```
Comment (Ana06) [inline src/api/.rubocop.yml]: The supported styles are not needed

File: src/api/.rubocop.yml [hunk 2]
```diff
@@ -65,6 +65,14 @@ Style/HashSyntax:
   Exclude:
     - 'config/routes.rb'
 
+# Checks the indentation of the first element in an array
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
```
Comment (Ana06) [inline src/api/.rubocop.yml]: The configuration parameters are not needed

File: src/api/.rubocop.yml [hunk 3]
```diff
@@ -65,6 +65,14 @@ Style/HashSyntax:
...
+# SupportedStyles: special_inside_parentheses, consistent, align_brackets
+Style/IndentArray:
+  Enabled: true
+  EnforcedStyle: consistent
```
Comment (Ana06) [inline src/api/.rubocop.yml]: why using `consistent` and no the default one?

File: src/api/.rubocop.yml [hunk 4]
```diff
@@ -65,6 +65,14 @@ Style/HashSyntax:
...
+  IndentationWidth: 2
```
Comment (Ana06) [inline src/api/.rubocop.yml]: Why is this needed?

**Comment by marcus-h** [comment]
diff --git a/src/api/config/initializers/hoptoad.rb b/src/api/config/initializers/hoptoad.rb
index 7709522..732d76a 100644
--- a/src/api/config/initializers/hoptoad.rb
+++ b/src/api/config/initializers/hoptoad.rb
@@ -9,17 +9,17 @@

...

-  ActivXML::Transport::NotFoundError
...
+    ActivXML::Transport::NotFoundError

While you're at it: this should probably be "ActiveXML..."

File: src/api/.rubocop.yml (hunk 3 above)
Comment (DavidKang) [inline src/api/.rubocop.yml]: I like it more, but I can put it by default if you prefer

Comment (Ana06) [inline src/api/.rubocop.yml]: Talking about styling I always prefer the 'standard' if there is not a good reason for changing. And in this case I personally prefer the default too, I think it is more readable. @openSUSE/open-build-service what others think? That would be the difference:

``` Ruby
# defaul
array = [
  :value
]
but_in_a_method_call([
                       :its_like_this
                     ])
# consistent
array = [
  :value
]
and_in_a_method_call([
  :no_difference
])
```

**Comment by ChrisBr** [comment]
LGTM
